src/xdocs/config_coding.xml

```diff
@@ -4897,6 +4897,96 @@ if (&quot;something&quot;.equals(x))
       </subsection>
     </section>
 
+    <section name="UnnecessarySemicolonInEnumeration">
+      <p>Since Checkstyle 8.22</p>
+      <subsection name="Description" id="UnnecessarySemicolonInEnumeration_Description">
+        <p>
+            Checks if unnecessary semicolon is in enum definitions.
+            Semicolon is not needed if enum body is empty.
```

Comment:
> if enum body is empty

According to https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.9 ,
The body includes the enumeration definitions as well as everything after the `;`?
Should this say something like `if the enum body is empty or contains only enum constants`.

Reply:
Technically speaking, if enum body is empty there should be no semicolon at well, since it is a part of enum constants declaration.
So it is only for cases where we have only enum constants in enum body (there can be 0 constants, only semicolon)

I corrected it, please take a look.